sys/include/xtimer.h

```diff
@@ -47,7 +47,7 @@ extern "C" {
  * @note This is a struct in order to make the xtimer API type strict
  */
 typedef struct {
-    uint64_t ticks64;
+    uint64_t ticks64;       /**< Captain Obvious to the rescue! */
```

Comment:
I do not object jokes -on the contrary- for instance in comments on TODOs, FIXMEs or really ugly hacked code parts, I think they are often more then appropriate. But for (doxygen) code documentation IMHO its better to have it brief (and to the point) first, and funny second. Hence, _timer ticks_ should suffice here, though its also very obvious, but doxygen is pedantic, as always.